@here/olp-sdk-core/lib/utils/DataStoreDownloadManager.ts

```diff
@@ -103,9 +103,8 @@ export class DataStoreDownloadManager implements DownloadManager {
                 retryCount >= maxRetries
             ) {
                 if (
-                    response.status === STATUS_CODES.OK ||
-                    response.status === STATUS_CODES.NO_CONTENT ||
-                    response.status === STATUS_CODES.CREATED
+                    response.status >= STATUS_CODES.CONNTINUE &&
+                    response.status < STATUS_CODES.BAD_REQUEST
```

Comment:
Not sure if 3xx codes should be considered here. E.g. 301. You can try as long as you want but you will always get the same response back and you should use the HTTP Location response header to get the moved to address for the next call. Is this going to help the user in any way if he doesn't know about it?

Reply:
Updated, but also 302 added. The response of a create, update or delete catalog can be 302